### DISCUSSION TITLE
The pysgems cannot get project 'pjt' and do not know how to run Indicator_Kriging.

### DISCUSSION THREAD

**Discussion by 984002717** [robinthibaut/pysgems#125] [Q&A]
I used pysgems to run kriging or sgsim algorithms in Folder 'examples',but I cannot get the project ‘’pjt‘ ’in the working directory, and  the most impotant is that I do not know why? Is anybody run the example projects successfully? Would you tell where am I Wrong？and thanks for that.
Besides, Is anybody run indicator kriging successfully? And would you share 'indicator_kriging.xml' file to me?  and thanks for that.

**Comment by GuillaumeVandekerckhove** [comment]
Hello, thank you for your interest. Did you follow the steps to install SGeMS and add it to your windows path? If so, is it possible to upload a printscreen of the error you get?
For the indicator kriging part there were still some issues, but those are resolved now and the version is updated.
